### PR TITLE
Deploy Prometheus & node-exporter

### PR DESCRIPTION
**Pull request by jchristgit**
To start off, we are only scraping Prometheus itself and node-exporter.

image

This needs some firewall changes (separately), see #54.

### CODE REVIEW

File: host_vars/ritchie/prometheus.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  alerting:
+    alertmanagers:
+      - static_configs:
+          - targets: []
+
+  rule_files:
+    # - "first_rules.yml"
+    # - "second_rules.yml"
+
+  scrape_configs:
```
Comment (ChrisLovering) [inline host_vars/ritchie/prometheus.yml:18]: Would this be where we add other services that need to be scraped?

When we get the VPN up and running, is this where we'd add the services running on other machines too?

Comment (jchristgit) [reply host_vars/ritchie/prometheus.yml:18]: Yes

**Review by ChrisLovering — APPROVED** [review]
Looks fine to me, I still haven't got personal infra setup to test it though.

**Comment by jchristgit** [comment]
Thanks!

**Pull request merged**